tests/test_quantum_info_metrics.py

```diff
@@ -238,3 +240,48 @@ def test_process_fidelity(backend):
     )
     backend.assert_allclose(gate_error(channel, backend=backend), 0.0)
     backend.assert_allclose(gate_error(channel, channel, backend=backend), 0.0)
+
+
+def test_entangling_capability(backend):
+    nqubits = 2
+    samples = 500
+
+    c1 = Circuit(nqubits)
+    c1.add([gates.RX(0, 0, trainable=True) for i in range(nqubits)])
```

Comment:
Here and in the other test function you're adding `I` twice to the same qubit before and after the `CNOT`. Why?